src/test/java/com/redhat/devtools/lsp4ij/features/selectionRange/TypeScriptSelectionersTest.java

```diff
@@ -0,0 +1,1128 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ * Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+
+package com.redhat.devtools.lsp4ij.features.selectionRange;
+
+import com.redhat.devtools.lsp4ij.fixtures.LSPSelectionersFixtureTestCase;
+
+/**
+ * TypeScript test for {@link LSPCodeBlockStatementGroupSelectioner} and {@link LSPCodeBlockBodySelectioner}.
+ */
+public class TypeScriptSelectionersTest extends LSPSelectionersFixtureTestCase {
```

Comment:
TypeScript-based test that confirms the expected behavior of the statement group and code block selectioners.